pkg/controllers/management/drivers/nodedriver/machine_driver.go

```diff
@@ -136,6 +173,37 @@ func (m *Lifecycle) download(obj *v3.NodeDriver) (*v3.NodeDriver, error) {
 	if err != nil && !errors.IsAlreadyExists(err) {
 		return obj, err
 	}
+	obj, err = m.createCredSchema(obj, credFields)
+	if err != nil {
+		return obj, err
+	}
+
+	return obj, nil
+}
+
+func (m *Lifecycle) createCredSchema(obj *v3.NodeDriver, credFields map[string]v3.Field) (*v3.NodeDriver, error) {
+	name := obj.Spec.DisplayName + "credentialconfig"
```

Comment:
just to confirm: is it ok the schema name to be lowercased? `credentialconfig`. Also check if display name of the node driver can be replaced with the actual schema name of the node driver

Reply:
yeah, it's ok, display name of node driver is created from the node driver in a similar way, 
`dynamicSchema.Name = obj.Spec.DisplayName + "config"`